### ISSUE TITLE
.NET Foundation Campaign: Jessica White

### ISSUE THREAD

**Issue by jessicabrentnall** [dotnet-foundation/election#86]
### The TL;DR.

* I offer the valuable, but rarely heard, perspective of those without multiple years of experience and contribution. From my time running diverse events, training junior and student developers, and from public speaking I’ve demonstrated that I can articulate sensitive and complex concepts, actively listen to others, speak up for them and act with empathy.

* I want to make it easier and more enjoyable to contribute to .NET projects for all.

* I’m passionate about encouraging diversity and empowerment.

## Who Am I?

Hi, I’m Jessica. I am a back-end software engineer, community advocate, conference speaker and conference organiser.

I actively partake in advocacy and outreach, not solely around .NET, but about technology as an industry. In the past I founded a diversity meetup, which has successfully run monthly events for four years and outgrown two venues. 

I often speak, mentor and volunteer at schools, colleges, universities and bootcamps. Further to this, I am currently running a technology conference which is not language specific, having a focus on making the event inclusive and enjoyable.

## Why I’m Running

The .NET Foundation has been open about its efforts to lower the barrier to entry to OSS and .NET related events. I know how intimidating it can be to make those initial contributions to projects, and how difficult it can be to find the support and guidance that give you the confidence to make those first steps. 

I feel it is important that people of all levels and backgrounds feel empowered and informed on how best to contribute. Contributors should be safe in the knowledge that feedback  will be provided in a kind and constructive way.

I joined the industry late, coming from a non-typical background, and in many respects I am still quite new. I see this as a strength because I represent a largely unheard from group that we want to encourage to take more of an active part in .NET.

If we include the input of different backgrounds, experiences and disciplines, we can make not just .NET, but all technology stronger.

## My .NET Open Source Contributions

* Have actively help set up and run OSS projects for my past employers, including introducing the standards proposed by GitHub and addressing any issues and pull requests. 

* Have actively given feedback on how I believe the experience can be improved for those who are new to raising issues or pull requests on the Microsoft repositories.

* Recently raised an issue on the Visual Studio Test Platform repository.

* Had a pull request accepted on the repository for the documentation for ASP.NET and ASP.NET Core.

## Links
* [My blog:](https://jesswhite.co.uk)

* [My Notist page:](https://noti.st/jesspwhite) 

**Links relating to DDD East Midlands - the conference I've co-founded**

* [DDD East Midlands Conference Blog Page](https://medium.com/dddeastmidlands)

* [DDD East Midlands Conference Webpage](https://www.dddeastmidlands.com/)

## Contact Information

* Twitter: [@JessPWhite](https://twitter.com/JessPWhite)

* Github: [@wordshaker](https://github.com/wordshaker)

* LinkedIn: [Jessica White](https://www.linkedin.com/in/jessica-white-67917883/)
